### DISCUSSION TITLE
I got error while trying to access Landsat 2 data from 2021-12-01 to 2022-01-01

### DISCUSSION THREAD

**Discussion by paulosabayomi** [microsoft/PlanetaryComputer#153] [General]
The date
![image](https://user-images.githubusercontent.com/68705275/209244340-88f4aecc-7d7b-4bde-9746-0d9a5403380d.png)

the error
![image](https://user-images.githubusercontent.com/68705275/209244317-32c935ee-085b-4cab-9c47-8d065774105a.png)

and if I try to get data in 2022 it returns 0 results, is the data not available in 2022 yet or the current year data is not available until the next year

**Comment by TomAugspurger** [comment]
Do you have a full copy-pasteable example I can run?
________________________________
From: Abayomi Amusa ***@***.***>
Sent: Thursday, December 22, 2022 5:54:54 PM
To: microsoft/PlanetaryComputer ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [microsoft/PlanetaryComputer] I got error while trying to access Landsat 2 data from 2021-12-01 to 2022-01-01 (Discussion #153)


The date
[image]<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F68705275%2F209244340-88f4aecc-7d7b-4bde-9746-0d9a5403380d.png&data=05%7C01%7Ctom.augspurger%40microsoft.com%7C88f1dbdea47548e5991b08dae477ecfd%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638073500975173339%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=LNI5TrP7FeTgipWhIuXkkxMK2zugap0WYaBIjvco9BI%3D&reserved=0>

the error
[image]<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F68705275%2F209244317-32c935ee-085b-4cab-9c47-8d065774105a.png&data=05%7C01%7Ctom.augspurger%40microsoft.com%7C88f1dbdea47548e5991b08dae477ecfd%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638073500975173339%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=OKEzrMugFsAEhAmA76TQCSxhjHSTevD2IPS4K4YdNTE%3D&reserved=0>

and if I try to get data in 2022 it returns 0 results, is the data not available in 2022 yet or the current year data is not available until the next year
